### PR TITLE
CE-3569: Close modal after building infobox

### PR DESCRIPTION
**Pull request by lukaszkonieczny**
Close infobox builder dialog after clicking back or publish button

### CODE REVIEW

File: extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxBuilderDialog.js [hunk 1]
```diff
@@ -16,6 +16,10 @@
 ve.ui.WikiaInfoboxBuilderDialog = function VeUiWikiaInfoboxBuilderDialog(config) {
 	// Parent constructor
 	ve.ui.WikiaInfoboxBuilderDialog.super.call(this, config);
+
+    $(window).on('infoboxBuilderReturnToVE', function () {
```
Comment (Grunny) [inline extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaInfoboxBuilderDialog.js]: Spaces -> tabs.

File: extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js [hunk 2]
```diff
@@ -32,6 +32,10 @@ define('wikia.infoboxBuilder.ponto', ['wikia.window', 'ponto'], function (w, pon
 			w.history.back();
 			return true;
 		};
+
+		this.returnToVE = function () {
+			$(window).trigger('infoboxBuilderReturnToVE');
+		}
```
Comment (Grunny) [inline extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js]: Should have a semi-colon.

File: extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js [hunk 3]
```diff
@@ -32,6 +32,10 @@ define('wikia.infoboxBuilder.ponto', ['wikia.window', 'ponto'], function (w, pon
 			w.history.back();
 			return true;
 		};
+
+		this.returnToVE = function () {
+			$(window).trigger('infoboxBuilderReturnToVE');
```
Comment (Grunny) [inline extensions/wikia/PortableInfoboxBuilder/js/PortableInfoboxBuilderPonto.js:37]: This module should depend on jQuery now too.

**Comment by adamkarminski** [comment]
LGTM

**Pull request merged**